test/Microsoft.AspNet.Server.KestrelTests/ConnectionFilterTests.cs

```diff
@@ -80,7 +81,36 @@ public class ConnectionFilterTests
                         "Hello World!");
                 }
             }
-       }
+        }
+
+        [ConditionalFact]
+        [FrameworkSkipCondition(RuntimeFrameworks.Mono, SkipReason = "Test hangs after execution on Mono.")]
```

Comment:
We no longer need this skip condition, right? Though I guess we should remove them all in a separate commit.